src/modules/ROOT/pages/changelog/202303/05.json

```diff
@@ -0,0 +1,161 @@
+{
+  "meta": {
+    "releasedOn": "2023.08.11"
+  },
+  "added": [
+    {
+      "what": [
+        "option to configure whether module field value is on the same row as it's label."
+      ],
+      "why": [
+        "to provide the user with the option to set the position of a field value."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1339"
+      ]
+    },
+    {
+      "what": [
+        "case sensitive search in a record list if MySQL database is used."
```

Comment:
I think it was changed it so it's no longer case sensitive (should be case insensitive now)